packages/readable-js-assets-webpack-plugin/test/fixtures/webpack.config.js

```diff
@@ -17,9 +17,7 @@ module.exports = {
 		concatenateModules: true,
 		minimizer: [
 			new TerserPlugin( {
-				cache: true,
 				parallel: true,
-				sourceMap: true,
```

Comment:
@jsnajdr, is it correct? I had to remove both flags because they were removed in webpack 5. The output generated in the tests needs to be verified, too.

Reply:
Yes, removing both flags is correct IMO. According to the `terser-webpack-plugin` [5.0.0 release notes](https://github.com/webpack-contrib/terser-webpack-plugin/releases/tag/v5.0.0), these options are now inherited from the parent webpack config.